Sofa/Component/AnimationLoop/src/sofa/component/animationloop/MultiTagAnimationLoop.cpp

```diff
@@ -131,7 +131,7 @@ void MultiTagAnimationLoop::step(const sofa::core::ExecParams* params, SReal dt)
     }
     sofa::helper::AdvancedTimer::stepEnd("UpdateMapping");
 
-    if (!SOFA_NO_UPDATE_BBOX)
```

Comment:
you don't remove the macro from the CMakeLists?

Reply:
the preproc macro is still used in BaseAnimationLoop, to set the default value of d_computeBoundingBox